knowledge-base/how-to-change-the-culture-of-the-datepicker-in-html5-mvc-report-viewer.md

```diff
@@ -0,0 +1,61 @@
+---
+title: Change the culture of Kendo DatePicker widget in the parameters area of the HTML5 MVC Report Viewer
+description: This KB article gives a brief description of how to set another language of the Kendo date picker in HTML5 MVC Report Viewer.
```

Comment:
remove the beginning, as it is not informative and will take the place of the informative content that will be pushed. 'How to set another language of the Kendo date picker in HTML5 MVC Report Viewer.' should do